q2lint.py

```diff
@@ -38,6 +38,9 @@ def main():
                 errors.append("Package dependencies should be stored in a "
                               "conda recipe instead of setup.py "
                               "`install_requires`.")
+            if ('license=\'BSD-3-Clause\',' not in text and
```

Comment:
This is a minor readability suggestion, feel free to ignore:

Instead of escaping quote characters, you can enclose each string in either double or single quotes, depending on what quote characters exist in the string:

```python
if "license='BSD-3-Clause'" not in text and 'license="BSD-3-Clause"' not in text:
    ...
```

The escape characters were tripping me up a little -- I didn't know that you could escape `"` inside of a single-quoted string literal! IOW, `'"foo"' == '\"foo\"'` :confused: